### PR TITLE
remove eslint and done responsiveness in student-correction-form

### PR DESCRIPTION
**Pull request by priyalpepcus**

### CODE REVIEW

File: src/dataGrid.css [hunk 1]
```diff
@@ -591,21 +576,21 @@ html, body {
...
-    position: relative;
-    height: 400px;
-    max-height: 100%;
-    width: 350px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    background-size: cover;
-}
+/*.student-registration-wrapper {*/
```
Comment (avi-jain-pepcus) [inline src/dataGrid.css]: Why we have commented code here?

File: src/dataGrid.css [hunk 2]
```diff
@@ -857,7 +835,7 @@ body,
     top: 0;
     margin: 0;
     z-index: 2;
-    padding: 0px 0;
+    padding: 0 0;
```
Comment (avi-jain-pepcus) [inline src/dataGrid.css]: padding: 0 0; is same as padding: 0;

File: src/dataGrid.css (hunk 1 above)
Comment (priyalpepcus) [reply src/dataGrid.css]: Removed.

File: src/dataGrid.css (hunk 2 above)
Comment (priyalpepcus) [reply src/dataGrid.css]: Replaced padding:0 0;  by padding: 0;

**Pull request merged**